pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -1,44 +1,45 @@
-{ stdenv, fetchurl, pkgconfig, which
+{ stdenv, fetchFromGitHub, pkgconfig
 , boost, libtorrentRasterbar, qtbase, qttools, qtsvg
 , debugSupport ? false # Debugging
 , guiSupport ? true, dbus_libs ? null # GUI (disable to run headless)
 , webuiSupport ? true # WebUI
 }:
 
-assert guiSupport -> (dbus_libs != null);
```

Comment:
If you remove this assertion, you should also make `dbus_libs` non-optional above; and please rename them to `dbus`.

Reply:
Yes. No way to avoid assertion here.

Reverted.